packages/prisma/src/scripts/2024-02-01_migrate_learning_elements.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  ElementInstanceType,
+  ElementOrderType,
+  ElementStackType,
+  ElementType,
+  LearningElement,
+  QuestionInstance,
+} from 'dist'
+import { PrismaClient } from '../client'
+
+function preparePracticeQuizInstanceOptions(
+  learningElement: LearningElement,
+  questionInstance: QuestionInstance
+) {
+  if (
+    questionInstance.questionData.type === ElementType.SC ||
+    questionInstance.questionData.type === ElementType.MC ||
+    questionInstance.questionData.type === ElementType.KPRIM ||
+    questionInstance.questionData.type === ElementType.NUMERICAL ||
+    questionInstance.questionData.type === ElementType.FREE_TEXT
+  ) {
+    return {
+      pointsMultiplier: questionInstance.pointsMultiplier,
+      resetTimeDays: learningElement.resetTimeDays,
+    }
+  }
+
+  return {}
+}
+
+function preparePracticeQuizElementData(
+  learningElement: LearningElement,
+  questionInstance: QuestionInstance
+) {
+  return {
+    ...questionInstance.questionData,
+  }
+}
+
+function preparePracticeQuizInstanceResults(
+  learningElement: LearningElement,
+  questionInstance: QuestionInstance
+) {
+  return {
+    ...questionInstance.results,
```

Comment:
Do we want to keep the results in the current form and migrate them later to a more general setup, where the total number of answers is tracked as well, etc.? This change would probably also require backend changes, so it might sense to postpone it, but track the task to be completed before the v3.1 release